client/client.go

```diff
@@ -116,6 +119,7 @@ func getRawQuoteViaDevice(d Device, reportData [64]byte) ([]uint8, error) {
 // getRawQuoteViaProvider use QuoteProvider to fetch quote in byte array format.
 func getRawQuoteViaProvider(qp QuoteProvider, reportData [64]byte) ([]uint8, error) {
 	if err := qp.IsSupported(); err == nil {
+		logger.V(1).Info("Get raw quote via QuoteProvider")
```

Comment:
Changing the log to "Get raw TDX quote via QuoteProvider" would make it more readable.

Reply:
Done.